index.html

```diff
@@ -2,14 +2,36 @@
 <html>
     <head>
         <title>Nauczyciele</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link rel="stylesheet" href="styles/style.css" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Ledger&family=Roboto&display=swap" rel="stylesheet">
     </head>
     <body>
-        <h1>Nauczyciele</h1>
-        <div>
-            <button>Kamil Cyganik</button><br>
-            <button>Izabela Heller</button><br>
-            <button>Daniel Pochaba</button><br>
-            <button>Przemysław Kłonica</button><br>
+        <header class="header">
+            <svg width="20" height="23" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
You can give the image an id="[id name]" parameter, then in style.css call it with #[id name] and add margin-left attribute to make arrow's position more like in figma.